docs/dev/replay-e2e-test.md

```diff
@@ -0,0 +1,49 @@
+# Replay End-to-End Tests
```

Comment:
we should add link to this doc from dev_steup.md

Reply:
Yes.